docs/datasets.md

```diff
@@ -88,6 +88,8 @@ The `Advanced` view lets you edit the underlying JSON configuration directly.
 
 ![Dataset Editing: General Tab](images/dataset_general.png)
 
+{% embed url="https://www.youtube.com/watch?v=yYauIHZcI_4 "%} 
```

Comment:
I'm not sure whether this video fits here. The documentation is about importing/managing datasets, whereas the video is more of an advertisement for our dataset alignment services (which is not really documenting any wK functionality). I can imagine it to be at the very bottom (or top?) with a caption "By the way, we also offer dataset alignment services, click [here](scm.io/...) to find out more."

What do you think @hotzenklotz?

Reply:
I agree with @daniel-wer. Have they "advertisement" in separate sentence makes sense.